src/nvim/eval.c

```diff
@@ -11745,28 +11665,31 @@ static void f_jobwait(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     return;
   }
 
+
   list_T *args = argvars[0].vval.v_list;
-  list_T *rv = tv_list_alloc();
+  Channel **jobs = xcalloc(sizeof(*jobs), args->lv_len);
```

Comment:
For the result it does not matter, but the arguments of `xcalloc()` are switched.